tables/study_table.tsv

```diff
@@ -1,4 +1,3 @@
-studyID	pubMedID	trait	reportedTrait	citation	altmetricScore	ethnicity	superPopulation	initialSampleSize	replicationSampleSize	sex	pValueAnnotation	betaAnnotation	ogValueTypes	numAssociationsFiltered	title	lastUpdated
```

Comment:
Should this header be gone? I can't remember if we require them

Reply:
Oh, that should still be there! Good catch!